src/Symfony/Component/PasswordHasher/README.md

```diff
@@ -0,0 +1,14 @@
+PasswordHasher Component
+========================
+
+The PasswordHasher component provides password hashing utilities for both old and modern algorithms,
+along with hash migration capabilities to keep applications on top of security standards.
+
```

Comment:
Please add a "Getting Started" section like https://github.com/symfony/translation#getting-started (I'll submit a PR for the components that don't have this yet).